README.md

```diff
@@ -18,11 +36,19 @@ namespace ConsoleApp1
         static void Main(string[] args)
         {
             var services = new ServiceCollection();
-            services.AddSingleton<ICommandHandler, MoveCommandHandler>();
-            services.AddCLI();
+            services.AddLogging(builder =>
```

Comment:
I copied this code to new .NET Core Console app and it does not build:

_ServiceCollection' does not contain a definition for 'AddLogging' and no accessible extension method 'AddLogging' accepting a first argument of type 'ServiceCollection' could be found (are you missing a using directive or an assembly reference?)_

Reply:
Agreed I will adjust that.